FahrplanDatenGarten/DBApis/hafasImport.py

```diff
@@ -1,33 +1,31 @@
 import datetime
 
-import pytz
-from pyhafas import HafasClient
-from pyhafas.profile import DBProfile
-
-from core.models import (Agency, Journey, JourneyStop, Source, StopID,
+from core.models import (Journey, JourneyStop, Provider, Source, StopID,
                          StopIDKind)
+from pyhafas import GeneralHafasError, HafasClient
+from pyhafas.profile import DBProfile
 
 
 class HafasImport:
     def __init__(self):
         self.hafasclient = HafasClient(DBProfile())
-        self.db, _ = Agency.objects.get_or_create(name="db")
-        self.dbapis, _ = Source.objects.get_or_create(name="dbapis")
-        self.idkind, _ = StopIDKind.objects.get_or_create(name='eva')
-        self.timezone = pytz.timezone("Europe/Berlin")
+        self.provider = Provider.objects.get(internal_name="db")
```

Comment:
Why don't you use `get_or_create` here?

Reply:
The idea is that the provider is already in the database because the dbapis_importstations was used. I can change it if you think using get_or_create would be better

Reply:
No, i think its ok